### ISSUE TITLE
some questions about `svtigs`

### ISSUE THREAD

**Issue by wjwei-handsome** [asylvz/SVarp#6]
Hi @asylvz , `svtigs` is cool representation, but I still have some  questions about it:

how to get the breakpoints or `call` variants? My process is aligning `svtigs.fa` to `ref.fa`, so that I could get the variants of `ref`.

But how about the non-reference path? After got the variants of `some_path`, how to augment the original graph by using this `non-reference` variants ?

Sincerely Looking forward to your answer :D

Best regards,
Wenjie

**Comment by asylvz** [comment]
You are right, if your reference does not have the related alternative sequences, then you can't find the non-reference breakpoints. This is the drawback of linear refs.

About augmentation, I don't know of any specific algorithm but here (https://www.biorxiv.org/content/10.1101/2024.04.18.590093v1.abstract), we use a pipeline to augment the graph. You can check this. Another alternative might be VG (https://github.com/vgteam/vg). They have graph augmentation as far as I know, but I'm not sure if it will work in your case.

**Comment by wjwei-handsome** [comment]
what do you think about that fill `svtigs.fa` to original graph ?

**Comment by asylvz** [comment]
I couldn't understand what you mean?

**Comment by wjwei-handsome** [comment]
sorry, what I mean is that augment the original graph by `svitg.fa`
like:
minigraph -cxggs <original-graph> <svtig.fa> -o  < augmented.gfa>
